_posts/2020-04-6-packaging-changes.md

```diff
@@ -0,0 +1,14 @@
+---
+title:  "Major Changes to Packaging Tool and Signing Taco Files"
+abstract: "The packaging tool no longer signs the .taco files in favor of the user calling jarsigner directly"
```

Comment:
user -> author

Reply:
> Packaging Tool 2.0 bring major changes to signing Taco files

Reply:
Version 2.0 implies new functionality, while this is technically the opposite. I hesitate to give it a number that high since there is no additive changes.

Reply:
Since we're following [semantic versioning](https://semver.org/) I _think_ that this qualifies as a major version change since it's a change that isn't backward compitible.